azure-pipelines-integration-dartlab.yml

```diff
@@ -29,6 +29,8 @@ variables:
   value: $(Build.SourcesDirectory)\artifacts\log\$(_configuration)
 - name: __VSNeverShowWhatsNew
   value: 1
+- name: RAZOR_RUN_FLAKY_TESTS
+  value: 'true'
```

Comment:
I thought you were talking about having a separate pipeline to run all tests? Seems like we're just guaranteeing failures if we turn this on here, which means guaranteeding people will have to investigate them, but we already have issues for these.

I would expect either a different pipeline for all tests, so we can compare the two, or use the `FlakyFact` attribute only on tests that we know for sure will at least pass some % of the time.

Reply:
I'm open for either. 

> I would expect either a different pipeline for all tests, so we can compare the two, or use the FlakyFact attribute only on tests that we know for sure will at least pass some % of the time.

This was my initial thought. Running the test pipeline and having failure rates generated is worth it. If we want a separate pipeline that's fine too, but I don't much see the point. I figure Conditional Skip = may be enabled at some point. It's the difference between "might work" and "definitely won't work". 0% success rate should just be disabled or fixed.

Reply:
I guess I agree with you, I'm just not confident that our existing skipped tests are all there because of flakiness, and not some other issue. We'll find out! 😀

Reply:
Well, we're off to a good start! 

![image](https://github.com/dotnet/razor/assets/475144/8cefd44c-1a22-4a6c-9a2d-112cdf64da8a)